File: sol/solution/0001-0100/0004. Median of Two Sorted Arrays/Solution.ex
defmodule Solution do
    @spec find_median_sorted_arrays(nums1 :: [integer], nums2 :: [integer]) :: float
    def find_median_sorted_arrays(nums1, nums2) do
        merged = Enum.sort(nums1 ++ nums2)
        len = length(merged)
        mid = div(len, 2)
        if rem(len, 2) == 1 do
            Enum.at(merged, mid) * 1.0
        else
            (Enum.at(merged, mid - 1) + Enum.at(merged, mid)) / 2.0
        end
    end
end

File: sol/solution/0001-0100/0004. Median of Two Sorted Arrays/Solution.rb
# @param {Integer[]} nums1
# @param {Integer[]} nums2
# @return {Float}
def find_median_sorted_arrays(nums1, nums2)
    s = (nums1+nums2).sort
    s.size.odd? ? s.at(s.size/2) : (s.at((s.size/2)-1)+s.at((s.size/2)))/2.0
end